stea/stea_keys.py

```diff
@@ -44,7 +44,8 @@ class SteaInputKeys:
     ECL_MULT = "mult"
     ECL_GLOB_MULT = "glob_mult"
     SERVER = "stea_server"
-    START_YEAR = "start_year"
+    START_DATE = "start_date"
+    START_YEAR = "start_year"  # Deprecated
```

Comment:
I am a bit confused - is the plan for `START_YEAR` to stay for now with deprecation warning and we delete it later?

Reply:
Yes.